Gemfile.lock

```diff
@@ -499,38 +503,38 @@ GEM
     omniauth-rails_csrf_protection (1.0.1)
       actionpack (>= 4.2)
       omniauth (~> 2.0)
-    pagy (3.6.0)
-    parallel (1.19.2)
-    parallel_tests (2.21.1)
+    pagy (3.14.0)
+    parallel (1.24.0)
+    parallel_tests (2.32.0)
       parallel
     parser (2.7.2.0)
       ast (~> 2.4.1)
-    path_expander (1.0.4)
-    pg (1.1.3)
-    pry (0.10.3)
+    path_expander (1.1.1)
+    pg (1.5.4)
+    pry (0.12.2)
       coderay (~> 1.1.0)
-      method_source (~> 0.8.1)
-      slop (~> 3.4)
+      method_source (~> 0.9.0)
     pry-byebug (3.3.0)
       byebug (~> 8.0)
       pry (~> 0.10)
-    pry-rails (0.3.5)
-      pry (>= 0.9.10)
-    pry-rescue (1.4.5)
+    pry-rails (0.3.9)
+      pry (>= 0.10.4)
+    pry-rescue (1.6.0)
       interception (>= 0.5)
-      pry
+      pry (>= 0.12.0)
     pry-stack_explorer (0.4.9.3)
       binding_of_caller (>= 0.7)
       pry (>= 0.9.11)
-    public_suffix (5.0.1)
+    public_suffix (5.0.4)
```

Comment:
### [public_suffix](https://simonecarletti.com/code/publicsuffix-ruby)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 5.0.1 → 5.0.4

[_[change-log](https://github.com/weppos/publicsuffix-ruby/blob/master/CHANGELOG.md), [source-code](https://github.com/weppos/publicsuffix-ruby/tree/v5.0.4), [gem-diff](https://my.diffend.io/gems/public_suffix/5.0.1/5.0.4)_]





<details>
<summary>Commits</summary>

A change of **54** commits. See the full changes on [the compare page](https://github.com/weppos/publicsuffix-ruby/compare/v5.0.1...v5.0.4).

These are the first 10 commits:
- (3fe4612) [Updated list from source (#222)](https://github.com/weppos/publicsuffix-ruby/commit/3fe461275c5a1cacf840154beb514a5af2f2e883)
- (bc53ab7) [Add Ruby 3.2 to the CI matrix (#223)](https://github.com/weppos/publicsuffix-ruby/commit/bc53ab7ef1f11ed649bd47f1c26b802fdd79efed)
- (668bda5) [Updated list from source (#224)](https://github.com/weppos/publicsuffix-ruby/commit/668bda53f8270ff96324e8dd70893613c9c98687)
- (3f811b3) [Updated list from source (#225)](https://github.com/weppos/publicsuffix-ruby/commit/3f811b328bd747a3a0e8878ffab2cad2411f32a9)
- (69a78ae) [Updated list from source (#226)](https://github.com/weppos/publicsuffix-ruby/commit/69a78ae5b52c66fd043d2e3b1b934b111ba0ef4a)
- (d9d921e) [Updated list from source (#227)](https://github.com/weppos/publicsuffix-ruby/commit/d9d921e12433a855c8d9412574275d1f8096d380)
- (f0f0ac6) [Updated list from source (#228)](https://github.com/weppos/publicsuffix-ruby/commit/f0f0ac69c5aa7e5f5facaa0dd5f547ff9dd079de)
- (bb3cbc4) [Updated list from source (#229)](https://github.com/weppos/publicsuffix-ruby/commit/bb3cbc4e254ced35c3506a1aecfa5e4406b2e351)
- (f332e64) [Updated list from source (#231)](https://github.com/weppos/publicsuffix-ruby/commit/f332e64a82230608fc0d7a60b86cffaa9661ba50)
- (33daaef) [Updated list from source (#232)](https://github.com/weppos/publicsuffix-ruby/commit/33daaef189bf31b2b903cb8b8501ea96d30fd666)


</details>